app/backend/core/authentication.py

```diff
@@ -304,51 +324,38 @@ async def validate_access_token(self, token: str):
                         jwks = await resp.json()
 
         if not jwks or "keys" not in jwks:
-            raise AuthError({"code": "invalid_keys", "description": "Unable to get keys to validate auth token."}, 401)
```

Comment:
note: we are switching away from json errors here

Reply:
I looked through the file and there seemed to be an inconsistent mix of JSON and non-JSON errors. So I debated whether to json.dumps() or just pass a string. We could json.dumps() if you think that's the better choice.